src/main/java/de/unistuttgart/crosswordbackend/data/Roles.java

```diff
@@ -0,0 +1,7 @@
+package de.unistuttgart.crosswordbackend.data;
+
+import java.util.List;
+
+public class Roles {
+    public static final List<String> LECTURER_ROLE = List.of("lecturer");
```

Comment:
See my last comment in the other PR:
```suggestion
public final class Roles {
    public static final List<String> LECTURER_ROLE = List.of("lecturer");
    
    private Roles() {}
```